src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Fastly 
+description: Learn how to configure a Fastly CDN for Adobe Commerce using API Mesh.
+keywords:
+  - API Mesh
+  - Extensibility
+  - Cache
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Fastly configuration
+
+Using [Fastly](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/fastly.html?lang=en) as a content delivery network (CDN) with API Mesh for Adobe Developer App Builder provides additional security and improved performance. To fully integrate API Mesh, Fastly, and Adobe Commerce, you must follow the configuration instructions in the following sections.
+
+## Configure Fastly in Adobe Commerce
+
+Use the following process to configure edge caching for Adobe Commerce using the provided Fastly CDN.
+
+1. [Get your Fastly credentials](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/setup-fastly/fastly-configuration.html?lang=en#get-fastly-credentials).
```

Comment:
You will need to get your provided credentials (Fastly Service ID** and **Fastly API Token )which usually come included with Adobe Commerce